.github/workflows/ci-benchmark.yaml

```diff
@@ -37,6 +37,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - run: rm -f test-connections.yaml
+      - run: ls -la # temporary to debug
```

Comment:
It may be worth changing this PR to draft until we change the current directory to be `python-sdk`.
The other change we discussed, of having a consistent way to override the env vars placeholders on the YAML (to make it more similar to `conftest.py`)  is optional.

Reply:
@tatiana is it ok if I can take this in separate PR as improvement because I don't think will get time today to test it created an issue to track it https://github.com/astronomer/astro-sdk/issues/936

Reply:
@tatiana is it ok if I can take this in separate PR as an improvement because I don't think will get time today to test it created an issue to track it https://github.com/astronomer/astro-sdk/issues/936